portfolio/src/main/webapp/index.html

```diff
@@ -9,16 +9,78 @@
   <body>
     <div id="content">
       <h1>Leah's Portfolio</h1>
-      <p>Hi, I'm Leah. I'm a rising Junior Computer Science student at North Carolina A&T State University, but I grew up in Prince George's County, Maryland. </p>
-      <img src = "/images/me.JPG" alt = "Picture of me :)" width = "400" height = "600">
-      <p>This is my portfolio.</p>
-      <p>Click here for a fact about me:</p>
-      <button onclick="addRandomFact()">Click Here</button>
+      <p>Hi, I'm Leah. I'm a rising Junior Computer Science student at North Carolina A&T State University, 
+          but I grew up in Prince George's County, Maryland. </p>
+      <img src = "/images/me.JPG" alt = "Picture of me :)" width = "450" height = "550" class = "center">
+      <p>This is my personal portfolio.</p>
+      <p>Click below for a fact about me:</p>
+      <button onclick="addRandomFact()">click here</button>
       <div id="fact-container"></div>
       <br>
       <a href="https://www.linkedin.com/in/leahmayo/">Let's connect on LinkedIn!</a>
       <br>
       <a href="https://github.com/leah-mayo">Check out my GitHub!</a>
+      
+      <h2> Comp Sci Courses I've Taken</h2>
+      <p>Intro to Computer Programming</p>
+      <p>Computer Programming Design</p>
+      <p>Discrete Structures</p>
+      <p>Data Structures</p>
+      <p>Analysis of Algorithms</p>
+
+      <h2>Hidden Talents</h2>
+      <p>I have always been the quiet type, and I've never really liked being in a huge spotlight. 
+          So, I surprised everyone, including myself, last year when I wrote my first song, recorded it, and released it to SoundCloud. It wasn't surprising to me
+          that I had the technical abilities to figure out how to use Garageband to produce and record. It wasnt even surprising to me that my song had a catchy tune; I've been 
+          singing my whole life, really. It was surprising because I finally let my pent-up creativity flow freely and put it out there for the world to see, 
+          after a lifetime of putting it on the back burner. </p>
+
+      <!-- Header for the Photo Grid --> 
+      <div class="header" id="myHeader">
+        <h1>My Photo Gallery</h1>
+        <p>Click on the buttons to change the grid view.</p>
+        <button class="btn" onclick="one()">1</button>
+        <button class="btn" onclick="two()">2</button>
+        <button class="btn" onclick="four()">4</button>
+      </div>
+      <!-- Photo Grid pictures situated in rows and columns --> 
+      <div class="row">
+        <div class="column">
+            <img src="/images/me2.HEIC" alt= "me" style="width:100%">
```

Comment:
You could create a css class for 

.image-width-fullsize{
width: 100%;
}
and add the class to the img.
This will make it easier if you want to update the width or add a border later.

Reply:
The way it is now, the image depends on the "style" word because that's how the sizes are changed when the buttons are clicked. Could I possibly add style to this class as well or am I thinking about it in the wrong way?

Reply:
I see what you are saying. So you could go about this two ways: 

1.) The current way of directly modifying the style attribute on the element in javascript onclick. 

Or

2.) Adding or removing classes to your img element on button click. 
https://www.w3schools.com/howto/howto_js_add_class.asp
https://www.w3schools.com/howto/howto_js_remove_class.asp

Number 2 is future proofing for any changes. It allows you to control all the CSS style in a CSS file rather than modifying in javascript. The created CSS classes can be re-used in multiple places. Last if you decide to use width 98% instead 100% or align: center, you only need to update the css and not the js.

Let me know if that doesn't make sense and we can have a quick sync where I can show you an example.

Reply:
That makes complete sense I'm going to try number 2, Ill let you know if I have any trouble